scripts/lib/CIME/XML/generic_xml.py

```diff
@@ -3,12 +3,18 @@
 be used by other XML interface modules and not directly.
 """
 from CIME.XML.standard_module_setup import *
+import xml.etree.ElementTree as ET
 from distutils.spawn import find_executable
 import getpass
 import six
 
 logger = logging.getLogger(__name__)
 
+class _Element(object): # private class, don't want users constructing directly
```

Comment:
Returning XML Elements as this class will prevent users from making their own Element or making calls directly on those elements (notice there are no methods on this class).